src/main/markdown/doc/latest/tutorial/style.md

```diff
@@ -582,7 +582,6 @@ You've already applied an HTML class attribute to the cell element to right-alig
 ```
 // Populate the Price and Change fields with new data.
     stocksFlexTable.setText(row, 1, priceText);
-    <span class="strike">stocksFlexTable.setText(row, 2, changeText + " (" + changePercentText + "%)");
```

Comment:
I think this should still be in there but should actually read:

stocksFlexTable.setText(row, 2, changeText + " (" + changePercentText + "%)");